Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1465,6 +1465,30 @@ public RelativeVigorIndex RVI(Symbol symbol, int period, MovingAverageType movin
             return relativeVigorIndex;
         }
 
+        /// <summary>
+        /// Creates a new RollingSharpeRatio indicator.
+        /// </summary>
+        /// <param name="symbol">The symbol whose RSR we want</param>
+        /// <param name="sharpePeriod">Period of historical observation for sharpe ratio calculation</param>
+		/// <param name="riskFreeRate">Risk-free rate for sharpe ratio calculation</param>
```

Comment:
It looks like you have been using tabs instead of spaces, so this is misaligned. Could you please review all the files in this pull-request and replace tabs for spaces?

Reply:
Hi Alex,

Thank you for reviewing my code! I will certainly address the changes listed. 

As for the _counter variable, I spoke with Jared regarding the math and why it is necessary to be implemented. Essentially since we are using the ROC indicator (which utilizes the current and prior values), and taking the SMA of it, we will be using values of a period "SharpePeriod + 1." Since the SMA and _sharpeRatio indicators have a period of "SharpePeriod," we need this counter to tell the indicator that we need one more iteration beyond the SharpePeriod-th iteration for the indicator to be fully warmed up. If we don't implement it, the indicator's first value (after being declared ready), will be incorrect.

Hopefully this clears it up!